dist.ini

```diff
@@ -0,0 +1,60 @@
+; REF: Dist::Zilla https://metacpan.org/pod/Dist::Zilla
+name = Crypt-OpenSSL-X509
+author = Dan Sully
+main_module = X509.pm
+license = Perl_5
+copyright_holder = Dan Sully
+copyright_year = 2004-2021
+
+; REF: https://metacpan.org/pod/Dist::Zilla::PluginBundle::Filter
+[@Filter]
+-bundle = @Basic
+-remove = GatherDir
+-remove = MakeMaker
+
+; REF: https://metacpan.org/pod/Dist::Zilla::Plugin::GatherDir
+[GatherDir]
+; this plugin needs no config and gathers most of your files
+exclude_filename = LICENSE
+exclude_filename = Makefile.PL
+
+; REF: Dist::Zilla::Plugin::GithubMeta: https://metacpan.org/pod/Dist::Zilla::Plugin::GithubMeta
+[GithubMeta]
+homepage = https://github.com/dsully/perl-crypt-openssl-x509
+issues = 1
+
+; REF: Dist::Zilla::Plugin::ReadmeAnyFromPod: https://metacpan.org/pod/Dist::Zilla::Plugin::ReadmeAnyFromPod
+; REF: http://blogs.perl.org/users/ovid/2014/03/a-simple-distini-for-distzilla.html
+[ReadmeAnyFromPod / MarkdownInRoot]
+filename = README.md
+
+; REF: Dist::Zilla::Plugin:::VersionFromMainModule https://metacpan.org/pod/Dist::Zilla::Plugin::VersionFromMainModule
+[VersionFromMainModule]
+
+; REF: Dist::Zilla::Plugin::ExtraTests https://metacpan.org/pod/Dist::Zilla::Plugin::ExtraTests
+[ExtraTests]
+
+; REF: Dist::Zilla::Plugin::PodSyntaxTests: https://metacpan.org/pod/Dist::Zilla::Plugin::PodSyntaxTests
+[PodSyntaxTests]
+
+; REF: Dist::Zilla::Plugin::Test::Kwalitee: https://metacpan.org/pod/Dist::Zilla::Plugin::Test::Kwalitee
+[Test::Kwalitee]
+
+; REF: Dist::Zilla::Plugin::MetaJSON: https://metacpan.org/pod/Dist::Zilla::Plugin::MetaJSON
+[MetaJSON]
+
+; REF: Dist::Zilla::Plugin::Test::CPAN::Meta::JSON: https://metacpan.org/pod/Dist::Zilla::Plugin::Test::CPAN::Meta::JSON
+[Test::CPAN::Meta::JSON]
+
+; REF: https://metacpan.org/pod/Dist::Zilla::Plugin::Prereqs::FromCPANfile
+[Prereqs::FromCPANfile]
+
+; REF: https://metacpan.org/pod/Dist::Zilla::Plugin::CopyFilesFromBuild
+[CopyFilesFromBuild]
+copy = Makefile.PL
+copy = LICENSE
+
```

Comment:
I like some of the git plugins like:

[Git::GatherDir]
exclude_filename = cpanfile
exclude_filename = Makefile.PL
exclude_filename = dev-bin/cpanm
exclude_filename = Dockerfile
exclude_filename = MANIFEST
exclude_filename = README

[Git::Tag]
tag_format  = %V       ; this is the default
tag_message = %V       ; this is the default

[Git::Commit]
changelog = Changes      ; this is the default

I also add gpg signatures to all my modules now:

[Signature]